xonsh/pyghooks.py

```diff
@@ -278,6 +280,63 @@ def color_by_name(name, fg=None, bg=None):
     return tok, fg, bg
 
 
+@lazyobject
+def PYGMENTS_MODIFIERS():
+    # pygments doesn't support all modifiers.
+    # use None to represent unsupported
```

Comment:
I know pygments doesn't  support all the modifiers but prompt toolkit support `reverse` and `blink`. So maybe there was a way to use them still.

Reply:
Yeah, I think we might want to have a separate module for computing PTK 2 colors directly from color names.